public/app/features/serviceaccounts/state/actions.ts

```diff
@@ -16,6 +16,27 @@ export function loadServiceAccount(id: number): ThunkResult<void> {
   };
 }
 
+export function deleteServiceAccountToken(id: number): ThunkResult<void> {
+  return async (dispatch, getStore) => {
+    const serviceAccount = getStore().serviceAccountProfile.serviceAccount;
+    getBackendSrv()
```

Comment:
We can use await since it's async function.
```suggestion
    await getBackendSrv().delete(`${BASE_URL}/${serviceAccount.userId}/tokens/${id}`);
     dispatch(loadServiceAccountTokens());
```